docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -52,20 +75,28 @@ The WordPress [wp-env package](https://www.npmjs.com/package/@wordpress/env) let
 
 The `wp-env` package requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other versions of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
 
-After confirming that the prerequisites are installed, install `wp-env` globally from the command-line using:
+After you have installed Docker, go ahead and install `wp-env` globally from the command-line using:
 
 ```sh
 npm -g install @wordpress/env
 ```
 
-This will be used in the next [Plugin step](wp-plugin.ms). To confirm it is installed and available, run:
+To confirm it is installed and available, run:
 
 ```sh
 wp-env --version
-> 1.4.0
+> 1.6.0
 ```
 
-### Alternatives
+The `wp-env` script is used to create a Docker WordPress environment, to start use:
+
+```sh
+wp-env start
+```
+
+You can access your environment in your browser at: [http://localhost:8888/], the default username is `admin` and default password is `password`. For more information controlling the Docker enivironment see the [@wordpress/env package readme](https://github.com/WordPress/gutenberg/blob/master/packages/env/README.md).
+
```

Comment:
I believe we have a missing step as I have gone through the various steps and the last step I did was 
`wp-env start` mentioned in my comment above.

Reply:
I'll update to make clear it should be a plugin, when moved out of the tutorial that part got lost. 👍

Reply:
Updated. Let me know if that clarifies

Reply:
It did not clarify, as the next step is still missing. There are two options that confuse. There should not be an option. There should just be a next step is add .wp-env.json to terminal to....

As the user is here to setup a local WordPress dev site. Developing a plugin or contributing to WordPress/Gutenberg does not matter at this stage as the only thing we are doing is creating a foundation to develop from. At the end of this setup your WordPress development site tutorial then the user can be guided to Create a Block, Contribute to Gutenberg, Contribute to WordPress etc tutorials. There is no need as I see it to mention it earlier. Creating a foundation should be the same for all of them. One needs to be guided step by step with the most used/broad option to setup the foundation.